### DISCUSSION TITLE
Is is fine to manually update pixi?

### DISCUSSION THREAD

**Discussion by Yura52** [prefix-dev/pixi#2071] [Q&A] [answered]
Hi! On my system, because of technical issues on my side, I cannot use `pixi self-update` or package managers to update my installation of `pixi`. However, I can manually download the latest `pixi` binary and place it instead of the current one. Is this manual procedure a correct way to update an existing `pixi` installation without breaking anything?

**Comment by wolfv** [comment, marked as answer]
Hi @Yura52 yes, that should indeed work without issues. You might have to do something like `chmod u+x ./pixi` to make it executable (depending on the system).

**Reply by Yura52** [reply]
> Hi @Yura52 yes, that should indeed work without issues. You might have to do something like `chmod u+x ./pixi` to make it executable (depending on the system).

Great, thanks!
